content/resources/introduction-to-pulumi/index.md

```diff
@@ -0,0 +1,96 @@
+---
+# Name of the webinar.
+title: "Introduction to Pulumi"
+meta_desc: In this workshop, you will learn the fundamentals of Infrastructure as Code through a series of guided exercises using Pulumi's Cloud Engineering platform.
+
+# A featured webinar will display first in the list.
+featured: false
+
+# If the video is pre-recorded or live.
+pre_recorded: false
+
+# If the video is part of the PulumiTV series. Setting this value to true will list the video in the "PulumiTV" section.
+pulumi_tv: false
+
+# The preview image will be shown on the list page.
+preview_image: ""
+
+# Webinars with unlisted as true will not be shown on the webinar list
+unlisted: false
+
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: true
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: false
+block_external_search_index: false
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "introduction-to-pulumi"
+
+# The content of the hero section.
+hero:
+    # The title text in the hero. This also serves as the pages H1.
+    title: "Introduction to Pulumi"
+    # The image the appears on the right hand side of the hero.
+    image: "/icons/containers.svg"
+
+# Webinar pages support multiple session via the 'multiple' property.
+multiple:
+    - datetime: 2020-11-16T09:30:00-07:00
```

Comment:
think this needs to be 2020-11-17T10:30:00-0700

Reply:
👍